logstash-core/build.gradle

```diff
@@ -108,10 +108,11 @@ dependencies {
     compile 'org.apache.logging.log4j:log4j-api:2.6.2'
     compile 'org.apache.logging.log4j:log4j-core:2.6.2'
     runtime 'org.apache.logging.log4j:log4j-slf4j-impl:2.6.2'
-    compile 'com.fasterxml.jackson.core:jackson-core:2.7.4'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.7.4'
-    compile 'com.fasterxml.jackson.module:jackson-module-afterburner:2.7.4'
-    compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.7.4'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.7.3'
```

Comment:
I can place a comment in here.  @robbavey - you are correct we need to stay in sync with the Java Jackson version used by Guy's JRuby wrapper gem.